lessons/03-html-lists/index.html

```diff
@@ -0,0 +1,67 @@
+<html>
+    <head>
+        <title>Hogwarts</title>
+     </head>
+     <body>
+         <h1>Hogwarts</h1>
+         <h2>Houses</h2>
+         <ul>
+             <li><strong>Gryffindor</strong></li>
+             <li>Hufflepuff</li>
+             <li>Ravenclaw</li>
+             <li>Slytherin</li>
+             <title>Notable teachers and staff members</title>
+         </ul>
+         <h3>Notable teachers and staff members</h3>
+         <ol>
+             <li>Argus Filch</li>
+               <li>Filius Flitwick</li>
+                <li>Gilderoy Lockhart</li> 
+               <li>Minerva McGonagall</li>  
+                <li>Poppy Pomfrey</li>
+                <li>Quirinus Quirrell</li>
+                <li>Horace Slughorn</li>
+                <li>Pomona Sprout</li>
+                <li>Sybill Trelawney</li>
+         </ol>
+         <ul>
+            <h4><li>Hogwarts ghosts</li></h4> 
+            </ul>
+            <ol>
+                <li>The Bloody Baron</li>
+                <li>The Fat Friar</li>
+                <li>The Grey Lady</li>
+                <li>Nearly Headless Nick</li>
+            </ol>
+            <ul>
+                   <h5><li>Hogwarts founders</li></h5>
+            </ul>
+            <ol>
+                <li>Godric Gryffindor</li>
+                <li>Helga Hufflepuff</li>
+                <li>Rowena Ravenclaw</li>
+                <li>Salazar Slytherin</li>
+            </ol>
+            <dl>
+                <dt><strong>List of spells</strong></dt>
+                <dd> Aberto</dd>
+                        <dd> A spell presumably used to open objects such as doors or windows.</dd>
+                <dd> Accio</dd>
+                <dd> This charm summons an object to the caster, potentially over a significant distance.</dd>
+                <dd>Expecto Patronum</dd>
+                <dd> Evokes a partially-tangible positive energy force known as a Patronus</dd>
+                <dd> Sectumsempra</dd>
+                <dd> A dark spell that creates large, blood-oozing gashes on the subject as if said subject had been "slashed by a sword".</dd>
+                <dd>Immobulus</dd>
+                <dd>Renders living targets immobile.</dd>
+                        <dd>Partis Temporus</dd>
+                        <dd>Creates a temporary gap through protective magical barriers.</dd>
+                        <dd>Locomotor</dd>
+                        <dd>The spell causes the named object to rise in the air and move around at the will of the caster.
+                        E</dd>
+                        </dl>
```

Comment:
Also close!

Here we need a title "List of spells" (hint: an h1 or h2 would work outside this list 😄)

Inside the `<dl>`  we use `<dt>` to indicate the title, and `<dd>` to indicate the description. Here it appears only `<dd>` is being used. Try changing the spell names to be in `<dt>` tags, but leave the spell descriptions in the `<dd>` tags.